modules/CbSystem/main/systemImpl.hpp

```diff
@@ -141,7 +141,7 @@ class systemImpl : public systemImplBase {
 
     /**
      *  @brief Checks if the firmware update marker is present and if so, 
-     *  notify the upper layer that the firmware update was successfuly installed.
+     *  notify the upper layer that the firmware update was successfully installed.
```

Comment:
Probably also "notifies", as the sentence before is also present tense.
```suggestion
     *  notifies the upper layer that the firmware update was successfully installed.
```